Guesser/FilterTypeGuesser.php

```diff
@@ -49,7 +49,9 @@ public function guessType($class, $property, ModelManagerInterface $modelManager
                     $options['operator_type'] = 'sonata_type_equal';
```

Comment:
I dont know how to fix that, not even what the problem is. Can you give more details?